Dockerfile

```diff
@@ -1,15 +1,12 @@
 FROM centos:centos7
 MAINTAINER Percona Development <info@percona.com>
 
-#ENV PACKAGE_URL https://repo.mysql.com/yum/mysql-5.6-community/docker/x86_64/mysql-community-server-minimal-5.6.28-2.el7.x86_64.rpm
-
 RUN rpmkeys --import https://www.percona.com/downloads/RPM-GPG-KEY-percona
 
 RUN yum install -y http://www.percona.com/downloads/percona-release/redhat/0.1-3/percona-release-0.1-3.noarch.rpm
-RUN yum install -y which Percona-Server-server-56 Percona-Server-tokudb-56
+RUN yum install -y which Percona-Server-server-56-5.6.28-rel76.1.el7 Percona-Server-tokudb-56-5.6.28-rel76.1.el7
```

Comment:
I was thinking along the lines of having predictable tags. Latest should always mirror our latest GA release and the tagged versions should follow the builds to avoid mixed version environments. Or are we not concerned by this?